aunt-leahs-app/src/components/adminLocationList/LocationList.jsx

```diff
@@ -117,7 +121,7 @@ export class LocationList extends React.Component {
 	}
 
 	async getLocations() {
-		const response = await fetch('http://localhost:7071/api/location');
+		const response = await authorizedFetch('/api/location', 'GET');
```

Comment:
Should we just use the volunteer api base url here since the GET request will available in the non-admin Functions app?

Reply:
Added